SimPEG/EM/Base.py

```diff
@@ -135,6 +148,23 @@ def MfMuiI(self):
             self._MfMuiI = self.mesh.getFaceInnerProduct(self.mui, invMat=True)
         return self._MfMuiI
 
+    # TODO: This should take a vector
+    def MfMuiIDeriv(self, u):
+        """
+        Derivative of :code:`MeSigma` with respect to the model
+        """
+        # TODO: only works for diagonal tensors. getEdgeInnerProductDeriv,
```

Comment:
can we raise an exception instead? There are tensor utils available to figure this out.